docs/source/science_examples.rst

```diff
@@ -0,0 +1,16 @@
+Science Examples
+=======================================
+
+.. toctree::
+  :maxdepth: 2
+  :caption: Science Examples:
+
+
+  tutorials/HLS/HLS_search.ipynb
```

Comment:
Same question from before, do we need to update the folder name to science_examples, or at least `science` since this is reflected in the urls? I don't think we're worried about breaking links at this point, so a url change should be ok.

Reply:
Yes this is a good time to change the naming conventions etc.

The folder names don't need to match the rst.  I seem to recall leaving things alone as much as possible.

Reply:
Speaking of which... how do we feel about `Science Examples` ?

Reply:
They don't need to match 100% but this is confusing because a different section has Tutorials in it's name. `science examples` is fine for now, I would make the folder `science` to make a nicer url.

Reply:
commit: 88ba9051f4c0be68f9c405affad6243c9f6e705c

renamed `tutorials/` to `science/`